README.md

```diff
@@ -1,27 +1,43 @@
 
 <h2>Please visit my wiki link for full list of questions</h2>
+
 <h3>https://github.com/mission-peace/interview/wiki</h3>
 
 <h2> Like my facebook page for latest updates on my youtube channel</h2>
 <h3>https://www.facebook.com/tusharroy25</h3>
 
 <h2> Contribution </h2>
-Please contribute to this repository to help it make better. Any change like new question, code improvement, doc improvement etc. is very welcome. Just send me a pull request and I will review the request and approve it if it looks good. 
+
+Please contribute to this repository to help it make better. Any change like new question, code improvement, docs improvement etc. is very welcome. Just send me a pull request and I will review the request and approve it if looks good. 
 
 <h2> How to use this repository </h2>
 
 <h3> Softwares to install </h3>
-* Install JDK8 https://docs.oracle.com/javase/8/docs/technotes/guides/install/install_overview.html
-* Install Git https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
-* Install either Intellij https://www.jetbrains.com/idea/download/
-* If you like eclipse instead of intellij install eclipse https://eclipse.org/downloads/
+
+- Install JDK8 https://docs.oracle.com/javase/8/docs/technotes/guides/install/install_overview.html
+
+- Install Git https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
+
+- Install either Intellij https://www.jetbrains.com/idea/download/
+
+- If you like eclipse instead of intellij install eclipse https://eclipse.org/downloads/
+
 
 <h3> Set up your desktop </h3>
-* Pull the git repository. Go to command line and type git clone https://github.com/mission-peace/interview.git
-* Go to root directory of checked out project.
-* Run ./gradlew idea to generate idea related classes
-* Fire up intellij. Go to Open. Go to git repo folder and open interview.ipr . On file menu go to project structure. Update language level support to 8
-* If you use eclipse, do ./gradlew eclipse . This will generate eclipse related files. Go to eclipse and open up folder containing this repo.
-* Go to any program and run that program
-* Go to any test and run the junit test.
-* Run ./gradlew build to create classes, run tests and create jar.
+
+- Pull the git repository. Go to command line and type 
+``git clone https://github.com/mission-peace/interview.git``
+
+- Go to root directory of checked out project.
+
+- Run ./gradlew idea to generate idea related classes
+
+- Fire up intellij. Go to Open. Go to git repo folder and open interview.ipr . On file menu go to project structure. Update language level support to 8
+
+- If you use eclipse, do ./gradlew eclipse . This will generate eclipse related files. Go to eclipse and open up folder containing this repo.
+
+- Go to any program and run that program
+
+- Go to any test and run the junit test.
+
+- Run ./gradlew build to create classes, run tests and create jar.
```

Comment:
could use `./gradlew build` since, this is done on line `29`  also so as to make it consistent? and also, for any command line or commands to be used